spec/models/appointment_spec.rb

```diff
@@ -0,0 +1,9 @@
+RSpec.describe Appointment, type: :model do
```

Comment:
```suggestion
RSpec.describe Appointment do
```

same here :nail_care: